lynis/package/install.sls

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- set sls_repo_file = tplroot ~ '.repo.install' %}
+{%- from tplroot ~ "/map.jinja" import lynis with context %}
+
+{% if lynis.use_repo %}
+include:
+  - {{ sls_repo_file }}
+{% endif %}
+
+lynis/package/install:
+  pkg.installed:
+    - pkgs:
+      - lynis
+      {% if lynis.repo.use_customers_repo and lynis.repo.install_plugins %}
+      - lynis-plugins
+      {% endif %}
+    {% if lynis.use_repo %}
+    - refresh_db: True
```

Comment:
I know `refresh_db` has been deprecated for `pkgrepo.managed` (replaced by `refresh`) and that's probably the same for `pkg` -- the [documentation](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html#salt.states.pkg.installed) appears to favour `refresh` as well.

Reply:
sure!

Reply:
done!